hack/tools/Makefile

```
@@ -65,13 +113,66 @@ KUSTOMIZE := $(BIN_DIR)/kustomize
 $(KUSTOMIZE): $(BIN_DIR) go.mod go.sum # Build kustomize from tools folder.
 	CGO_ENABLED=0 go build -tags=tools -o $@ sigs.k8s.io/kustomize/kustomize/v3
 
+MDBOOK_SHARE := $(SHARE_DIR)/mdbook$(MDBOOK_ARCHIVE_EXT)
+$(MDBOOK_SHARE): ../../versions.mk $(SHARE_DIR)
+	curl -sL -o $(MDBOOK_SHARE) "https://github.com/rust-lang/mdBook/releases/download/$(MDBOOK_VERSION)/mdBook-$(MDBOOK_VERSION)-x86_64-$(RUST_TARGET)$(MDBOOK_ARCHIVE_EXT)"
+
+MDBOOK := $(BIN_DIR)/mdbook
+$(MDBOOK): $(BIN_DIR) $(MDBOOK_SHARE)
+	$(MDBOOK_EXTRACT_COMMAND)
+	chmod +x $@
+	touch -m $@
+
+MDBOOK_EMBED := $(BIN_DIR)/mdbook-embed
+$(MDBOOK_EMBED): $(BIN_DIR) go.mod go.sum
+	go build -tags=tools -o $(BIN_DIR)/mdbook-embed sigs.k8s.io/cluster-api/hack/tools/mdbook/embed
+
+MDBOOK_RELEASELINK := $(BIN_DIR)/mdbook-releaselink
+$(MDBOOK_RELEASELINK): $(BIN_DIR) go.mod go.sum
+	go build -tags=tools -o $(BIN_DIR)/mdbook-releaselink sigs.k8s.io/cluster-api/hack/tools/mdbook/releaselink
+
+MDBOOK_TABULATE := $(BIN_DIR)/mdbook-tabulate
+$(MDBOOK_TABULATE): $(BIN_DIR) go.mod go.sum
+	go build -tags=tools -o $(BIN_DIR)/mdbook-tabulate sigs.k8s.io/cluster-api/hack/tools/mdbook/tabulate
+
 MOCKGEN := $(BIN_DIR)/mockgen
 $(MOCKGEN): $(BIN_DIR) go.mod go.sum # Build mockgen from tools folder.
 	go build -tags=tools -o $@ github.com/golang/mock/mockgen
 
 RELEASE_NOTES := $(BIN_DIR)/release-notes
 $(RELEASE_NOTES): $(BIN_DIR) go.mod go.sum
-	go build -tags tools -o $@ sigs.k8s.io/cluster-api/hack/tools/release
+	go build -tags tools -o $@ k8s.io/release/cmd/release-notes
```

```suggestion
	go build -tags tools -o $@ sigs.k8s.io/cluster-api/hack/tools/release
```
Please let's keep using our current release note tool.